static/src/javascripts/lib/geolocation.js

```diff
@@ -405,8 +405,17 @@ const extendedCurrencySymbol = {
     International: '$',
```

Comment:
TBD: Discuss what our 'real' default should be or if we are okay with this split in logic